dxp/build_process.md

```diff
@@ -72,6 +72,65 @@ The `liferay-npm-bridge-generator` is only executed when there is a `.npmbridger
 
 > To learn more about how to use this tool read ["How to use liferay npm bridge generator" in the wike of the liferay-js-toolkit repository](https://github.com/liferay/liferay-js-toolkit/wiki/How-to-use-liferay-npm-bridge-generator).
 
+### Build SCSS
+
+There are two ways that DXP handles building `.scss` files:
+
+-   [CSS Builder Gradle Plugin](https://github.com/liferay/liferay-portal/blob/master/modules/sdk/gradle-plugins-css-builder/README.markdown)
+-   Liferay npm bundler with [sass-loader](https://github.com/liferay/liferay-js-toolkit/tree/master/packages/liferay-npm-bundler-loader-sass-loader) and [css-loader](https://github.com/liferay/liferay-js-toolkit/tree/master/packages/liferay-npm-bundler-loader-css-loader) via JavaScript
+
+Gradle plugin is executed together with the `gradlew deploy` build pipeline and expects the `.scss` files in the module's `src/main/resources` folder. Compiled by the [`CSSBuilder`](https://github.com/liferay/liferay-portal/blob/master/modules/util/css-builder/src/main/java/com/liferay/css/builder/CSSBuilder.java) which is responsible for the build and [`CSSRTLConverter`](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-css/frontend-css-rtl-servlet/src/main/java/com/liferay/frontend/css/rtl/servlet/internal/converter/CSSRTLConverter.java) to create the [RTL](https://en.wikipedia.org/wiki/Right-to-left) files for the final css. The module portlet defines the path to the final CSS file that adds the CSS path to the HTML output of the request.
+
+Unlike the Gradle plugin, loaders are executed in the Liferay npm bundler, which consequently is part of the build flow of `liferay-npm-scripts`, the difference is that you can import a `.css` or `.scss` file in JavaScript file .
```

Comment:
This made me realize we don't do the `_rtl` conversion for these files... 🤔

Reply:
See https://github.com/liferay/liferay-frontend-guidelines/pull/209#discussion_r490036526 for some clarification... it's important to note that we go through `CSSBuilder` in any case since that's got us covered and then just use the `css-loader` on an already-processed file.

Phew! 😂

Reply:
haha